config/initializers/doorkeeper.rb

```diff
@@ -100,3 +100,4 @@
   # set to true if you want this to be allowed
   # wildcard_redirect_uri false
 end
+Doorkeeper.configuration.token_grant_types << "password"
```

Comment:
Could you remove this and edit line 86 to read the following instead?

```ruby
grant_flows %w(authorization_code password client_credentials)
```

Also, please rebase on top of master and add an entry to the changelog.